interface.js

```diff
@@ -0,0 +1,31 @@
+$(document).ready(function(){
+  var game = new Game
+  var frame = new Frame 
+
+  $('#bowl').click(function(){
+    // update()
+    game.bowl()
```

Comment:
The brief is to be able to record scores for 10 frames of a bowling game, not to generate random scores for the bowling. However given your implementation of the random functionality, this looks like good encapsulation